pyproject.toml

```diff
@@ -1,3 +1,268 @@
+[project]
+name = "sentry"
+version = "25.8.0.dev0"
+dependencies = [
+    "amqp==5.3.1",
+    "anyio==3.7.1",
+    "asgiref==3.8.1",
+    "attrs==24.2.0",
+    "avalara==20.9.0",
+    "beautifulsoup4==4.7.1",
+    "billiard==4.2.1",
+    "boto3==1.34.128",
+    "botocore==1.34.128",
```

Comment:
this doesn't seem right at all -- none of these should be pinned if this is the equivalent of requirements-base today

Reply:
i added these from requirements-frozen.txt - it was the easiest path to get an equivalent dependency tree - let me see if uv's resolver can get here from our top level deps